pmd-tsql/src/main/java/net/sourceforge/pmd/lang/tsql/TSqlLanguageModule.java

```diff
@@ -0,0 +1,28 @@
+/**
+ * BSD-style license; for more info see http://pmd.sourceforge.net/license.html
+ */
+
+package net.sourceforge.pmd.lang.tsql;
+
+import net.sourceforge.pmd.lang.BaseLanguageModule;
+
+/**
+ * Language Module for T-SQL.
+ * @deprecated There is no full PMD support for T-SQL.
+ */
+@Deprecated
+public class TSqlLanguageModule extends BaseLanguageModule {
```

Comment:
Please don't create this class at all - not even deprecated. We should not register a language as a PMD language, if it isn't...
Since this is a new module, we can directly leave it and don't need to repeat old mistakes.